cli/cmd/get.go

```diff
@@ -461,12 +461,13 @@ func describeModelInput(groupStatus *resource.APIGroupStatus, apiEndpoint string
 }
 
 func getAPISummary(apiEndpoint string) (*schema.APISummary, error) {
-	req, err := http.NewRequest("GET", apiEndpoint, nil)
+	httpsAPIEndpoint := strings.Replace(apiEndpoint, "http://", "https://", -1)
```

Comment:
do you expect more than one http in the url?

Reply:
nope! fixed